wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -81,6 +81,8 @@ public final class ZMUserSession: NSObject {
 
     var analyticsSession: (any AnalyticsSessionProtocol)?
 
+    var analyticsManager: (any AnalyticsManagerProtocol)?
```

Comment:
What is this needed?

Reply:
It is not needed anymore. Good catch: https://github.com/wireapp/wire-ios/commit/92cd5919408c490af8ff75cc74235101e008468b